geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/Frequency.scala

```diff
@@ -39,14 +39,17 @@ import scala.reflect.ClassTag
   * @param ct class tag
   * @tparam T type parameter, should match the type binding of the attribute
   */
-class Frequency[T] private [stats] (val sft: SimpleFeatureType,
```

Comment:
Did the 'private' modifier drop for reason?

Reply:
yeah, I made the constructor public as it's used in some other packages now, where creating a stat string first and then parsing it would be extra work. Originally the constructors were made private to encourage using the parser/stats API, but there isn't really any strong reason to keep them so.